Gemfile

```diff
@@ -23,7 +23,7 @@ gem 'http_accept_language'
 gem 'identity-doc-auth', github: '18F/identity-doc-auth', tag: 'v0.4.0'
 gem 'identity-hostdata', github: '18F/identity-hostdata', tag: 'v0.4.3'
 require File.join(__dir__, 'lib', 'lambda_jobs', 'git_ref.rb')
-gem 'identity-idp-functions', github: '18F/identity-idp-functions', ref: LambdaJobs::GIT_REF
+gem 'identity-idp-functions', '>= 0.11.0', github: '18F/identity-idp-functions', ref: 'e58edba856c9cb38e007b494d4f560c2039675b2'
```

Comment:
We could tag the commit and reference the tag instead.

Reply:
Yup! This is temporary while the other 2 PRs are in-flight